package.json

```diff
@@ -33,14 +33,15 @@
   "devDependencies": {
     "array-move": "^1.0.0",
     "array-range": "^1.0.1",
-    "babel-cli": "^6.2.0",
-    "babel-core": "^6.2.1",
-    "babel-eslint": "^7.1.1",
-    "babel-loader": "^6.2.4",
-    "babel-plugin-transform-flow-strip-types": "^6.22.0",
-    "babel-polyfill": "^6.2.0",
-    "babel-preset-es2015": "^6.18.0",
-    "babel-preset-react": "^6.5.0",
+    "babel-cli": "^6.24.0",
+    "babel-core": "^6.24.0",
+    "babel-eslint": "^7.2.1",
+    "babel-loader": "^6.4.1",
+    "babel-plugin-transform-runtime": "^6.23.0",
+    "babel-polyfill": "^6.23.0",
+    "babel-preset-env": "^1.3.2",
+    "babel-preset-flow": "^6.23.0",
+    "babel-preset-react": "^6.23.0",
```

Comment:
* added babel-plugin-transform-runtime to support async/await
* replaced babel-plugin-transform-flow-strip-types by babel-preset-flow
* upgraded babel because a newer plugin needed a newer babel.